Sources/Testing/Events/Clock.swift

```diff
@@ -338,3 +338,41 @@ extension Test.Clock.Instant {
 #endif
   }
 }
+
+@available(_clockAPI, *)
+extension Test.Clock.Instant: Codable {
+  private enum CodingKeys: CodingKey {
```

Comment:
Please document this type (a one-line description is fine) and name it `_CodingKeys` because it's `private`.